x-pack/test/observability_functional/apps/observability/alerts/index.ts

```diff
@@ -7,6 +7,13 @@
 
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { ObjectRemover } from '../../../../functional_with_es_ssl/lib/object_remover';
+import {
+  createAlert,
```

Comment:
@claudiopro In our code base I suggest we use the correct terminology
`{ import createAlert as createRule, disableAlert as disableRule, muteAlert as muteRule }`